src/apis/fetchData.ts

```diff
@@ -0,0 +1,6 @@
+export const fetchData = (url: string) => {
+  return fetch(url).then((response) => {
```

Comment:
Use shorthand.

Reply:
in the [last commit](https://github.com/zuko4celik/movies-app-new/pull/24/commits/2c9a79b44c2ed1a1bfa637b9d8af0a4b3fe064be).